README.md

```diff
@@ -108,43 +108,86 @@ See [CONFIG.md](CONFIG.md) for details.
 See [Code Samples](https://github.com/hazelcast/hazelcast-nodejs-client/tree/master/code_samples)
 
 
-# Serialization Considerations
+# Serialization
 
-Hazelcast needs to serialize objects in order to be able to keep them in the server memory. For primitive types, it uses Hazelcast native serialization. For other complex types (e.g. JS objects), it uses JSON serialization.
+Hazelcast serializes all your objects into an instance of `Data`. `Data` is the binary representation of an object. When Hazelcast serializes an object into `Data`, it first checks whether the object is an instance of `IdentifiedDataSerializable`, if not it checks if it is an instance of `Portable` and serializes it accordingly. For the following types Hazelcast optimizes the serialization a user can not override this behaviour.
```

Comment:
....Hazelcast optimizes the serialization and you cannot override this behavior

(notice the usage of "cannot" and "behavior").